src/main/java/org/folio/services/migration/async/AsyncMigrationJobService.java

```diff
@@ -44,6 +46,13 @@ public AsyncMigrationJobService(Context vertxContext, Map<String, String> okapiH
     this.migrationContext = new AsyncMigrationContext(vertxContext, okapiHeaders, postgresClient);
   }
 
+  public Future<AsyncMigrationJobCollection> getAllAsyncJobs() {
+    return migrationJobRepository.get(new Criterion())
+      .map(list -> new AsyncMigrationJobCollection()
```

Comment:
Since this returns all jobs, curious if there is some process to clean up old jobs periodically or it replies on the hosting to manage that?

Reply:
For now, there are no mechanisms to clean up old jobs. If it's necessary we can create a ticket for it.